runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollOverlay.java

```diff
@@ -85,6 +94,15 @@ public Dimension render(Graphics2D graphics)
 			}
 		}
 
+		if (clue.isRequiresLight() && plugin.getInventoryItems() != null)
+		{
+			if(!HAS_LIGHT.fulfilledBy(plugin.getInventoryItems()) || !HAS_LIGHT.fulfilledBy(plugin.getEquippedItems()))
+			{
+				panelComponent.getChildren().add(LineComponent.builder().left("").build());
+				panelComponent.getChildren().add(LineComponent.builder().left("Requires Light!").leftColor(Color.RED).build());
```

Comment:
```suggestion
				panelComponent.getChildren().add(LineComponent.builder().left("Requires Light Source!").leftColor(Color.RED).build());
```

Reply:
Could be worth using a different colour to make it distinct from the spade one, maybe with an orange